src/lib.rs

```diff
@@ -159,7 +161,18 @@ pub async fn load_query_file(world: &World, file_name: &str) -> Result<usize> {
         vec![],
         &*FN_REGISTRY.lock()?,
         &aggr_reg,
-    )?;
+    );
+    let query = match query {
+        Ok(query) => query,
+        Err(e) => {
+            let mut h = TermHighlighter::stderr();
+            if let Err(e) = Script::format_error_from_script(&raw, &mut h, &e) {
+                eprintln!("Error: {}", e);
```

Comment:
should this rather be logged?

Reply:
The error only occurs if the highlighter couldn't highlight, so in any likelihood, if we ever hit this, something is seriously wrong. so stderr is the most likely thing still working.

Reply:
I think the only time this error is hit is when the writer can't allocate memory

Reply:
Ah yeah! What i actually meant was that the highlighter is writing to stderr always, but we might not have an stderr around, depending on how tremor was started. But as long as we stop the program when this error hits, this is fine.